pageobjects/item.page.js

```diff
@@ -84,6 +92,14 @@ class ItemPage extends PageMixture {
 		return reference.$( this.constructor.ITEM_WIDGET_SELECTORES.PROPERTY_INPUT );
 	}
 
+	selectSuggestedProperty( propertyId ) {
+		const suggestionLabelSelector = this.constructor.GENERAL_SELECTORS.ENTITY_SUGGESTOR_SUGGESTION_LABEL;
+		const suggestionSelector = `${suggestionLabelSelector}[text() = "${propertyId}"]`;
```

Comment:
This fails if there is actually a Label set for this property. That might not be the case in CI, but it might very well be the case on developer machines, since this property can be reused with the `WIKIBASE_PROPERTY_STRING` environment variable.

Reply:
it feels because property suggester won't open? it should be exact same value too right?

Reply:
No, the suggester opens, but `text()` is not the Property ID, but the Property label and that the selector doesn't match anymore.

Reply:
oh yea I saw that in case of lexeme, where equality didn't match because of the id between parantheses being part of the label.

Another possible solution can be, as long as our tests keep the design of searching by id, is to just select the first suggested entry, like I did in ruby tests https://gerrit.wikimedia.org/r/c/mediawiki/extensions/Wikibase/+/541081/14/tests/browser/features/support/modules/reference_module.rb#19

I also had to use `starts-with` instead of equality in lexeme due to the id between parentheses addition to the entry text.

Another possible solution is to just select the first entry, which should work as long as our browser tests are designed to enter ids, making the first entry to be (always?) the desired entity.

What do you think? select on aliases or select first entry blindly?

Reply:
I'd go with the first one. Not nice but makes less assumptions and is just as error-prone as text matching. Case in point: Searching for Q11 in production.

Reply:
@wiest first one being selecting on aliases? or first one as selecting first suggested entity?

Reply:
Edited to clarify.

Reply:
Went for selecting first entry for now.. which also solves the problem with using WIKIBASE_PROPERTY_STRING (if hopefully it will always the first one in the suggestions depending on what value is used in it)